src/hooks/Recoil_Hooks.js

```diff
@@ -208,12 +218,18 @@ function useInterface(): RecoilInterface {
     }
 
     function useResetRecoilState<T>(recoilState: RecoilState<T>): Resetter {
+      if (__DEV__) {
+        validateRecoilState(recoilState);
+      }
       return () => setRecoilValue(storeRef.current, recoilState, DEFAULT_VALUE);
     }
 
     function useRecoilValueLoadable<T>(
       recoilValue: RecoilValue<T>,
     ): Loadable<T> {
+      if (__DEV__) {
+        validateRecoilState(recoilState);
+      }
```

Comment:
`useRecoilValueLoadable` is used by `useRecoilValue` which is used by `useRecoilState`